index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.css" />
+</head>
+<body>
+    <div id="container">
+        <header>
+            <span>My Logo</span>
+            <nav>
+                <div class="bar1"></div>
+                <div class="bar2"></div>
+                <div class="bar3"></div>
+            </nav>
+        </header>
+        <section class="headline">
+            <h2>I’m Ismail Glad to see you!</h2>
+            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <ul>
+                <li><i class="fa fa-twitter"></i></li>
+                <li><i class="fa fa-linkedin"></i></li>
+                <li><i class="fa fa-medium"></i></li>
+                <li><i class="fa fa-github"></i></li>
+                <li><i class="fa fa-angellist"></i></li>
+            </ul>
```

Comment:
- [ ] The requirement says that you design should stick to the Figma template it means Your website should look exactly like the design (e.g. font, colors, images, text, margins) ,
- [ ] Kindly Fix the color and the size of these icons 😄 to meet the requirement
- [ ] Also , you will notice with I that the margin bottom is different in your design , kindly fix that

| Your design | Figma design |
| :---: | :---: | 
| ![Capture 5](https://user-images.githubusercontent.com/80458199/234552158-05ec3bfc-f25d-455f-bcb0-758c1244551f.JPG) | ![Capture 7](https://user-images.githubusercontent.com/80458199/234552238-e7d6d9b1-7018-45ac-a49e-c1b10aa0f8c2.JPG) |